travis/maven-settings.xml

```diff
@@ -5,16 +5,16 @@
 
     <servers>
         <server>
-            <id>nexus.gadevs</id>
-            <username>${env.TRAVIS_NEXUS_USERNAME}</username>
-            <password>${env.TRAVIS_NEXUS_PASSWORD}</password>
+            <id>awsGeodesyOperations</id>
+            <username>${env.AWS_ACCESS_KEY_ID}</username>
+            <password>${env.AWS_SECRET_ACCESS_KEY}</password>
         </server>
     </servers>
     <mirrors>
         <mirror>
-            <id>nexus.gadevs</id>
-            <name>GA Nexus</name>
-            <url>https://nexus.gadevs.ga/repository/maven-public/</url>
+            <id>gnss-artifactory-release</id>
```

Comment:
It's not a mirror (it's not for all dependencies), it's just a place to store our own build artifacts.

Reply:
I see. So, same as igssitelog-binding project, both <server> and <mirror> will be removed as not needed.